lib/senders/email.js

```diff
@@ -267,50 +267,51 @@ module.exports = function (log, config) {
       optionalHeader('X-Uid', message.uid)
     )
 
-    var emailConfig = {
-      sender: this.sender,
-      from: this.sender,
-      to: message.email,
-      subject: localized.subject,
-      text: localized.text,
-      html: localized.html,
-      xMailer: false,
-      headers
+    let mailer = this.mailer
+    let emailService, emailSender
+    if (config.emailService.forcedEmailAddresses.test(message.email)) {
+      mailer = this.emailService
+      emailService = 'fxa-email-service'
+      emailSender = 'ses'
+    } else {
+      emailService = 'fxa-auth-server'
     }
 
-    // Utilize nodemailer's cc ability to send to multiple addresses
-    // Ref: https://nodemailer.com/message/
-    if (message.ccEmails) {
-      emailConfig.cc = message.ccEmails
-    }
+    // Set headers that let us attribute success/failure correctly
+    message.emailService = headers['X-Email-Service'] = emailService
+    message.emailSender = headers['X-Email-Sender'] = emailSender
 
     if (this.sesConfigurationSet) {
       // Note on SES Event Publishing: The X-SES-CONFIGURATION-SET and
       // X-SES-MESSAGE-TAGS email headers will be stripped by SES from the
       // actual outgoing email messages.
-      emailConfig.headers[X_SES_CONFIGURATION_SET] = this.sesConfigurationSet
+      headers[X_SES_CONFIGURATION_SET] = this.sesConfigurationSet
+      headers[X_SES_MESSAGE_TAGS] = sesMessageTagsHeaderValue(message.metricsTemplate || template, emailService)
```

Comment:
Previously, the code to set `X_SES_MESSAGE_TAGS` was copy/pasted separately at all of the callsites, even though they were all doing the same thing. I opted to move it inside this function to eliminate the repetition.

Doing that also necessitated introducing one small bit of conditional logic, because the expected value for `messageType` in the tests for `verifySyncEmail` is `verifyEmail`. Presumably that's for legitimate metrics reasons rather than an accident of history, so I kept the same behaviour by passing in an extra `metricsTemplate` property from that callsite and checking it here.